docs/source/code_reference/entry_variables.rst

```diff
@@ -0,0 +1,8 @@
+===============
+Entry Variables
+===============
+
+.. autoclass:: ytdl_sub.entries.script.variable_definitions.VariableDefinitions()
```

Comment:
I need to poke around and figure out how to get each property (variable) to show on the right-hand side - this page not very helpful otherwise